src/components/Synthetics/TradeBox/TradeBoxRows/AllowedSlippageRow.tsx

```diff
@@ -6,16 +6,28 @@ import { formatPercentage } from "lib/numbers";
 import ExchangeInfoRow from "components/Exchange/ExchangeInfoRow";
 import PercentageInput from "components/PercentageInput/PercentageInput";
 import TooltipWithPortal from "components/Tooltip/TooltipWithPortal";
+import { useSettings } from "context/SettingsContext/SettingsContextProvider";
+import {
+  selectSetTradeboxAllowedSlippage,
+  selectTradeboxAllowedSlippage,
+} from "context/SyntheticsStateContext/selectors/tradeboxSelectors";
+import { useSelector } from "context/SyntheticsStateContext/utils";
+import { useEffect } from "react";
+import { useTradeboxChanges } from "../hooks/useTradeboxChanges";
+
+export function AllowedSlippageRow() {
+  const { savedAllowedSlippage } = useSettings();
+  const allowedSlippage = useSelector(selectTradeboxAllowedSlippage);
+  const setAllowedSlippage = useSelector(selectSetTradeboxAllowedSlippage);
+
+  const tradeboxChanges = useTradeboxChanges();
+
+  useEffect(() => {
```

Comment:
is it correct place for such effect? maybe put it somewhere inside tradebox itself?